subsystems/kvm/ContainerFile

```diff
@@ -0,0 +1,32 @@
+# ContainerFile used to create the image available at quay.io/qm-images/kvm:latest
+#
+# How to build
+# ==================
+# podman login quay.io
+# podman build -t quay.io/qm-images/kvm:latest -f ContainerFile
+# podman push quay.io/qm-images/kvm:latest
+FROM fedora:latest
+
+ENV PASSWORD_FEDORA_USER=fedora
+
+RUN dnf -y install virt-install \
+                   libvirt-daemon \
+                   libvirt-daemon-qemu \
+                   libvirt-daemon-kvm \
+                   libvirt-daemon-config-network \
+                   guestfs-tools \
+                   wget \
+                   vim -y \
+    && dnf clean all && rm -rf /var/cache/dnf
+
+RUN wget -O /var/lib/libvirt/images/Fedora-Cloud-Base-Generic.qcow2 https://cofractal-ewr.mm.fcix.net/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2
```

Comment:
I would also use ENV_VAR for changing fedora mirror
Default should be 
```
https://download.fedoraproject.org/pub/fedora/linux/releases/
```

https://download.fedoraproject.org/pub/fedora/linux/releases/41/Cloud/x86_64/images/Fedora-Cloud-Base-Generic-41-1.4.x86_64.qcow2

Reply:
Hey @Yarboa Are you going to update this as well ?

Reply:
Prefer all be tested before merged

Reply:
I tested.